embeddings/get_started_with_embedding_tuning.ipynb

```diff
@@ -0,0 +1,1434 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #104.    def get_df_from_jsonl(file_path: str, nrows: int | None = None) -&gt; pd.DataFrame:
</pre><p> </p><p>Is <code>get_df_from_jsonl</code> really needed as a separate function? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
I prefer for code cleanliness